NuGet.Config

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="Repox Artifactory (proxy)" value="https://repox.jfrog.io/artifactory/api/nuget/nuget" />
+    <add key="Repox Artifactory (releases)" value="https://repox.jfrog.io/repox/api/nuget/sonarsource-nuget-releases" />
```

Comment:
hm do we need this at all? 
if we don't need the releases package source, I can close  https://github.com/SonarSource/sonar-scanner-msbuild/issues/1243